<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Kategori</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="labelKdKategori">
      <property name="text">
       <string>Kode Kategori</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="editKdKategori"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelNmKategori">
      <property name="text">
       <string>Nama Kategori</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="editNmKategori"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnSimpan">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>SIMPAN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHapus">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>HAPUS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUbah">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>UBAH</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBersih">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>BERSIH</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editCari">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>351</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Ketik untuk mencari...</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tabelKategori">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>351</width>
     <height>171</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Kode Kategori</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nama Kategori</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
